src/main/java/io/confluent/kafkarest/Context.java

```diff
@@ -48,4 +53,12 @@ public ProducerPool getProducerPool() {
   public ConsumerManager getConsumerManager() {
     return consumerManager;
   }
+
+  public SimpleConsumerFactory getSimpleConsumerFactory() {
```

Comment:
Looks like this isn't ever being accessed, I assume because it's only used by SimpleConsumerManager so nothing ever calls this method on Context. How about just removing the field from Context entirely?